.circleci/config.yml

```diff
@@ -67,14 +67,49 @@ commands:
       - run: make docker-watcher
       - run: make docker-watcher_info
 
-  install_elixir_and_check_docker_status:
+  check_docker_status:
```

Comment:
this step did too many things and was not usable from some of the jobs so I broken them down semantically